docs/guides/install/index.html

```diff
@@ -3,215 +3,248 @@
-        <td>
-          <a href="{{data.url.cdn}}/{{data.cdn.js.helixUi}}" target="_blank">
-            <hx-icon type="external-link"></hx-icon>
-            helix-ui.js
-          </a>
-        </td>
-        <td>N/A</td>
-      </tr>
-    </tbody>
-  </table>
-</section>
-
-<section><!-- Web Components -->
-  <h2 class="hxSectionTitle" id="web-components">Web Components</h2>
-  <section>
-    <table class="hxTable hxTable--condensed">
-      <thead>
-        <tr>
-          <th></th>
-          <th>Production (CDN)</th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr>
-          <th>ES5 Adapter</th>
-          <td>
-            <a href="{{data.url.cdn}}/{{data.cdn.js.es5adapter}}" target="_blank">
-              <hx-icon type="external-link"></hx-icon>
-              custom-elements-es5-adapter.js
-            </a>
-          </td>
-        </tr>
-        <tr>
-          <th>Web Components Loader</th>
-          <td>
-            <a href="{{data.url.cdn}}/{{data.cdn.js.polyfillLoader}}" target="_blank">
-              <hx-icon type="external-link"></hx-icon>
-              webcomponents-loader.js
-            </a>
-          </td>
-        </tr>
-      </tbody>
-    </table>
-
-    <p>
-      To begin using the custom elements defined in HelixUI, you must first
-      ensure that the browser can load them. To do so, you'll need to insert
-      both the ES5 Adapter and the Polyfill Loader into your markup.
-    </p>
+  <section><!-- Install from NPM -->
+    <h2 class="hxSectionTitle" id="install">Install from NPM</h2>
+    <p>Save to your devDependencies with the following command:</p>
+    {% code "bash" %}
+      npm install --save-dev helix-ui
+    {% endcode %}
   </section>
 
-  <section><!-- web components loader -->
-    <h3 class="hxSubSectionTitle" id="web-components-loader">Web Components Loader</h3>
-    <div class="hxAlert">
-      <div class="hxAlert__icon">
-        <hx-icon type="info-circle"></hx-icon>
-      </div>
-      <div class="hxAlert__text">
-        <span class="hxAlert__status">NOTE</span>
-        Currently, the loader only supports being loaded in the <code>&lt;head&gt;</code>
-        of an HTML document.
-      </div>
-    </div>
-    <br />
-
-    <p>
-      The web components loader comes from the
-      <a href="https://github.com/webcomponents/webcomponentsjs">webcomponentsjs</a>
-      project. Its job is to perform feature detection on the current browser
-      so that it can dynamically inject the smallest polyfill needed to
-      support web components into the head of the HTML document.
-    </p>
+  <section><!-- Select an Application Layout -->
+    <h2 class="hxSectionTitle" id="application-layout">Select an Application Layout</h2>
     <p>
-      Once the polyfills are loaded (if needed), the "WebComponentsReady" event
-      will be fired on the document to announce availability of Web Component
-      functionality.
+      You have two choices regarding an application layout.
```

Comment:
Here also - I would add a bit more context. What is an application layout, and why do I need to select one?